src/main/java/net/neoforged/neoforge/common/conditions/NeoForgeConditions.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common.conditions;
+
+import java.util.List;
+import net.minecraft.core.Registry;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.flag.FeatureFlag;
+import net.minecraft.world.flag.FeatureFlagSet;
+import org.apache.commons.lang3.ArrayUtils;
+
+public interface NeoForgeConditions {
```

Comment:
Make it a `public final class`, no?

Reply:
Could do that i just personally dont see the point, extending this class gives you no special benefits or methods, and being an interface lesses the amount of `public static [final]` needed everywhere as everything by default is `public static [final]`.

If preferred though i can change it to be a final class, i just personally dont see any reason for it to be a final class